src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2651,6 +2651,14 @@ private void CheckSequentialOnPartialType(BindingDiagnosticBag diagnostics)
                     }
                 }
             }
+
+            if (whereFoundField != null &&
+                PrimaryConstructor is { } primaryConstructor && primaryConstructor.GetCapturedParameters().Any() &&
+                (primaryConstructor.SyntaxRef.SyntaxTree != whereFoundField.SyntaxTree || primaryConstructor.SyntaxRef.Span != whereFoundField.Span))
+            {
+                diagnostics.Add(ErrorCode.WRN_SequentialOnPartialClass, Locations[0], this);
```

Comment:
```suggestion
                diagnostics.Add(ErrorCode.WRN_SequentialOnPartialClass, GetFirstLocation(), this);
```

to be consistent with https://github.com/dotnet/roslyn/pull/67678